scipy/linalg/_basic.py

```diff
@@ -141,11 +145,37 @@ def solve(a, b, lower=False, overwrite_a=False,
     array([ True,  True,  True], dtype=bool)
 
     """
+    xp = array_namespace(a, b)
+    if check_finite:
```

Comment:
I suppose the idea here is that it's easy enough for us to support `check_finite` even though it's not in the standard?

Not that we need to, but it seems like it would be just as easy to support `transposed`. With slightly more effort, we could support `lower` with other options for `assume_a`. 

Just want to make sure I'm understanding correct - not suggesting any action.

Reply:
Yes, and yes. I was just implementing what was 'free' in this PR (just using a kwarg of `_asarray` here) and leaving the rest in `xp_unsupported_args`. Ideally, that list would be reduced as much as possible, which can happen in two ways:
- us implementing agnostic versions of the args (which may be easy for some of them)
- args being added to the standard API